src/routes/safe/store/models/types/transactions.d.ts

```diff
@@ -0,0 +1,176 @@
+export enum Operation {
+  CALL,
+  DELEGATE_CALL,
+  CREATE,
+}
+
+export interface DecodedValue {
+  operation: keyof typeof Operation
+  to: string
+  value: number
+  data: string
+  decodedData: DataDecoded
+}
+
+export interface SingleTransactionMethodParameter {
+  name: string
+  type: string
+  value: string
+}
+
+export interface MultiSendMethodParameter extends SingleTransactionMethodParameter {
+  decodedValue: DecodedValue[]
+}
+
+export type Parameter = MultiSendMethodParameter | SingleTransactionMethodParameter
+
+export interface DataDecoded {
+  method: DataDecodedMethod
+  parameters: Parameter[]
+}
+
+export enum ConfirmationType {
+  CONFIRMATION,
+  EXECUTION,
+}
```

Comment:
are the initialisers left intentionally?

Reply:
sorry, I'm lost here.

How is that? What should have I done instead? help! :pleading_face:

Reply:
so if you omit the initializers for members of enum, the values would be 0, 1, 2, 3 etc. Was this done intentionally? Also I think we had a relevant discussion about enums in one of the others pr